config/config.exs

```diff
@@ -129,6 +129,12 @@ config :game_backend,
 # Configures Ecto migrations
 config :game_backend, GameBackend.Repo, migration_primary_key: [type: :binary_id]
 
+# {:ok, currency_config_json} =
+#   Application.app_dir(:game_backend, "priv/currencies_rules.json")
+#   |> File.read()
+
+# config :game_backend, :currencies_config, Jason.decode!(currency_config_json)
```

Comment:
Why the move and commented out?

Reply:
There was an error in the playtest, i'll restore this in this pr and fix it in a pr

Reply:
Restored!